### DISCUSSION TITLE
How long refresh token valid ??

### DISCUSSION THREAD

**Discussion by Noppadon1712** [adonisjs/core#2540] [Help] [answered]
Excuse me. 
Someone knows about refresh token default maxage in adonis v4 and how to config it ???
Thanks.

**Comment by McSneaky** [comment, marked as answer]
It doesn't expire

_I think you already got your answer from Discord, but responding in here too so others that land from search engines see it_

**Reply by Noppadon1712** [reply]
Thanks you.
